homeassistant/components/otbr/strings.json

```
@@ -8,6 +8,9 @@
         "description": "Provide URL for the OpenThread Border Router's REST API"
       }
     },
+    "progress": {
+      "connect_addon": "Connecting to Open Thread Border Router"
```

It's OpenThread as per other messages.

```suggestion
      "connect_addon": "Connecting to OpenThread Border Router"
```

However, we can also drop the word Open?